tfx/components/trainer/component.py

```diff
@@ -179,6 +180,7 @@ def trainer_fn(trainer.executor.TrainerFnArgs,
         that will be passed into user module.
       custom_executor_spec: Optional custom executor spec.
       output: Optional `Model` channel for result of exported models.
+      log_output: Optional `ModelRun` channel for the log output of models.
```

Comment:
not log only, might sth like 

"Optional `ModelRun` channel, as the working dir of models, can be used to output non-model related output, e.g., tensorboard logs"